### ISSUE TITLE
AFD Line breaks in VO

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1737]

**Comment by greg-does-weather** [comment]
Experimenting a bit, I wonder if those line breaks might just be how Jaws works. VoiceOver doesn't exhibit the same behavior on the AFDs I've looked at, and looking at the rendered HTML, there's no clear reason there should be any extraneous pauses. I did notice that they seemed to happen at about the time the page scrolled and I wonder if that's related or just coincidence.

**Comment by colinmurphy01** [comment]
Let's see if Dan can replicate this again using JAWS in our next manul review
